src/components/signTransaction/index.tsx

```diff
@@ -105,9 +128,19 @@ const SignTransaction = ({
       </p>
       <div>
         <button onClick={onSignTransactionClick}>Sign Transaction</button>
+        <button onClick={onSignTransactionRPC}>Sign Transaction RPC</button>
       </div>
     </div>
   );
 };
 
 export default SignTransaction;
+function createTransaction(arg0: {
+  network: BitcoinNetworkType;
+  paymentAddress: string;
+  ordinalsAddress: string;
+  paymentPublicKey: string;
+  ordinalsPublicKey: string;
+}): [any, any] | PromiseLike<[any, any]> {
+  throw new Error("Function not implemented.");
+}
```

Comment:
I think this may have been the template for the util which needs to be imported still 👀

Reply:
Good catch!